spec/rubocop/cop/rails/controller_testing_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::ControllerTesting, :config do
+  context 'Rails 4.2', :rails42, :config do
+    it 'does not add offense when extending ActionController::TestCase' do
+      expect_no_offenses(<<~RUBY)
+        class MyControllerTest < ActionController::TestCase
+        end
+      RUBY
+    end
+  end
+
+  it 'adds offense when extending ActionController::TestCase' do
+    expect_offense(<<~RUBY)
+      class MyControllerTest < ActionController::TestCase
```

Comment:
Nitpick: will it catch `::ActionController::TestCase`?

Reply:
It did not. I patched the node pattern to pick up on it. Thanks!!